### ISSUE TITLE
cannot import name 'States' from 'homeassistant.components.recorder.models

### ISSUE THREAD

**Issue by tkruger206** [KoljaWindeler/energy_calc#1]
Hi K-W, 

Thank you for making this great integration.  

I am seeing the following error in logs when trying to load the integration. I have tried using the HACS/GUI install and setup method as well as the manual install and setup. I see the same error with both. I see you made some edits a few months ago for "States", not sure if HA has changed things again or the issue you had persisted.  Appreciate any help here. Thanks. 

Logger: homeassistant.setup
Source: setup.py:298 
First occurred: 1:52:30 PM (1 occurrences) 
Last logged: 1:52:30 PM

Unable to prepare setup for platform energy_calc.sensor: Platform not found (cannot import name 'States' from 'homeassistant.components.recorder.models' (/usr/src/homeassistant/homeassistant/components/recorder/models.py)).

Full logs
2022-07-09 13:52:30 ERROR (MainThread) [homeassistant.setup] Unable to prepare setup for platform energy_calc.sensor: Platform not found (cannot import name 'States' from 'homeassistant.components.recorder.models' (/usr/src/homeassistant/homeassistant/components/recorder/models.py)).

**Comment by KoljaWindeler** [comment]
Hi, short fyi: I'm on it.
Fixed it some days ago but seeing issues. Have a running version that should also increase calculation accuracy.. testing tonight and tomorrow and will update if no more bugs appear

**Comment by tkruger206** [comment]
Das ist dope. Prost Kumpel. Du bist eine Legende.

**Comment by KoljaWindeler** [comment]
:) also bei mir tut das jetzt wieder .. guck mal bitte nach ob es bei dir auch der Fall ist

**Comment by Schorseg** [comment]
works for me as well

**Comment by tkruger206** [comment]
Works for me too.  

Is there a way to take the attributes from the sensor and make them as energy meter entity?  

I was looking to use the attribute output of this to feed into the energy dashboard for return to grid stats etc. 

Thanks,
Tim

**Comment by Schorseg** [comment]
Yes that does work. You can add it to the template: section and do it similar to this

> template:
 >  - sensor:
    - name: "grid2homekwh"
      unit_of_measurement: "kWh"
      state: "{{state_attr('sensor.energy_calc', 'Grid_to_home_[kWh]')}}"
      device_class: energy
      state_class: total

**Comment by tkruger206** [comment]
Ah ja, vielen Dank.
